algorithms/Java/linked-lists/fold-linked-list.java

```diff
@@ -0,0 +1,83 @@
+public static class Node {
+    int data;
+    Node next;
+}
```

Comment:
Node constructer is missing.

Reply:
Also, give a description of the question at top of the file before starting the program.